src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
   [issue](https://github.com/open-telemetry/opentelemetry-dotnet/issues/5092)
   for details and workaround.
   ([#5077](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5077))
+* Removed `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable support. The
```

Comment:
One question .. The grpc conventions are still not stable and this instrumentaion is still emitting them. Should we explicitly call it out, to avoid giving the feel that the entire conventions emitted by this is stable?

Reply:
I think it's fine, folks who want to spend time reading changelog should already know semver2.